JoinColony/colonyDapp
# Add MotionStaked event to ActionPage

Added MotionStaked event to feed of MintTokenMotion

**New stuff** ✨

* Added query and local resolver to get events for a specific motion
* Connected eventsForMotion query to MintTokenMotion page
* Added amountTag and staker event value to feed item

**How to test** (If #2429 isn't merged) :leaves:
1. Check out to this branch: `testing/backed-motion-event`. Which has the staking widget and event combined.
2. Start the stack
3. Get reputation
4. Create a motion
5. Deposit the tokens you are going to stake
6. Stake in the motion
7. Profit

![FireShot Capture 206 - Colony - localhost](https://user-images.githubusercontent.com/18473896/114745670-44e2d800-9d25-11eb-8325-0d77044d31dc.png)

Resolves DEV-271
